app/models/grda_warehouse/service_history_enrollment.rb

```diff
@@ -382,6 +382,33 @@ def self.service_types
     where(age: (18..Float::INFINITY))
   end
 
+  scope :cas_active_clients, -> do
```

Comment:
I think a further improvement would be for this scope to be used for the following methods: `project_group`, `boston`, `ce_with_assessment`, since those all have logic around scoping down ServiceHistoryEnrollments, that are implemented twice in 2 files.